### DISCUSSION TITLE
question about logging in with your account

### DISCUSSION THREAD

**Discussion by JunkyardCat** [cwendt94/espn-api#202] [Q&A]
can i use an account and pull data from the account? like its private and public league? if so how do i do it

**Comment by cwendt94** [comment]
Yes you can. If its a public league you will only need your `leagueId` for most of the features but it is recommended to pass your credentials to enable all features.

You can get your `leagueId` in ESPN's url that you use to view your league. For example this is a baseball league url https://fantasy.espn.com/baseball/team?leagueId=81134470&seasonId=2021&teamId=8. The `leagueId` is 81134470

Getting credentials look at this [discussion](https://github.com/cwendt94/espn-api/discussions/150)

After that you can access all of your league data. See [wiki](https://github.com/cwendt94/espn-api/wiki) for details on the sport API's

**Reply by JunkyardCat** [reply]
is there a way to get the cookies using username and password? im guessing no since it validates using a captcha?. is it possible to open the site using python then let the user login on that window then get the cookies then close the window?

**Reply by cwendt94** [reply]
I'm not sure if that is possible, but for this package that is a little more trouble then its worth. Getting the cookies from the browser doesn't take long and then they can be used for ever.
